src/clients/api/mutations/exitMarket.ts

```diff
@@ -1,6 +1,8 @@
+import { IAccount } from 'context/AuthContext';
+
 export interface IExitMarketInput {
   comptrollerContract: $TSFixMe; // @TODO: use contract type (through Typechain?)
-  account: string | undefined | null;
+  account: IAccount;
```

Comment:
I don't think we should change the implementation here, since we only need the address.
We can change the name and avoid using `null` eventually:
```suggestion
  accountAddress: string | undefined;
```

Reply:
done